client/blocks/product-selector/README.md

```diff
@@ -51,13 +51,24 @@ The following props can be passed to the Product Selector block:
 			monthly: [ 'jetpack_backup_daily_monthly', 'jetpack_backup_realtime_monthly' ],
 		}
 		```
+	* `optionDescriptions`: ( object ) Optional descriptions for the product options.
+	They replace a default `description` in case a given option (represented by the object's key) has been purchased.
+	Each key is a product slug, and the value is the corresponding copy. Example:
+		```
+		optionDescriptions: {
+			jetpack_backup_daily: __( 'Looking for more? Get unlimited real-time backup archives' ),
```

Comment:
Since this is Calypso, let's use `translate()` instead of `__()` for these.

Reply:
You're totally right! Somehow I misused it 🤦‍♂️